models.py

```diff
@@ -84,12 +165,32 @@ def weight_regularization_unet(model, regularize, weight_decay_conv2d):
 def callbacks_unet(callbacks_config):
     experiment_timing = ExperimentTiming(**callbacks_config['experiment_timing'])
     model_checkpoints = ModelCheckpoint(**callbacks_config['model_checkpoint'])
-    lr_scheduler = ExponentialLRScheduler(**callbacks_config['lr_scheduler'])
+    lr_scheduler = ExponentialLRScheduler(**callbacks_config['exp_lr_scheduler'])
+    training_monitor = TrainingMonitor(**callbacks_config['training_monitor'])
+    validation_monitor = ValidationMonitor(**callbacks_config['validation_monitor'])
+    neptune_monitor = NeptuneMonitorSegmentation(**callbacks_config['neptune_monitor'])
+    early_stopping = EarlyStopping(**callbacks_config['early_stopping'])
+
+    return CallbackList(
+        callbacks=[experiment_timing, training_monitor, validation_monitor,
+                   model_checkpoints, lr_scheduler, neptune_monitor, early_stopping
+                   ])
+
+
+def callbacks_dcan(callbacks_config):
```

Comment:
@apyskir You are repeating a lot here. Maybe have def callbacks_base() and both callbacks_unet and callbacks_dcan use this callbacks_base()

Reply:
Yes, seems like a nice idea.